.github/workflows/docs-deploy.yml

```diff
@@ -0,0 +1,47 @@
+name: Deploy to GitHub Pages
+
+on:
+  push:
+    branches:
+      - main
+      - 41-docs-website
```

Comment:
Is this going to stay or should it be removed before merging to main?

Reply:
Yes, removed